README.md

```diff
@@ -68,9 +68,9 @@ r
  ```
  sudo apt remove system76-pop-gtk-theme
  sudo make uninstall
- sudo rm -rf /usr/share/themes/{Pop,Pop-Eta,Pop-Nokto,Pop-Nokto-Eta}
- rm -rf ~/.local/share/themes/{Pop,Pop-Eta,Pop-Nokto,Pop-Nokto-Eta}
- rm -rf ~/.themes/{Pop,Pop-Eta,Pop-Nokto,Pop-Nokto-Eta}
+ sudo rm -rf /usr/share/themes/Pop{,-dark{,-slim},-light{,-slim},-slim{,-dark,-light}}
```

Comment:
These are actually leftover from when the theme was still based on Adapta. It covers the case that someone is using a very old (Version 1) copy of the theme. `sudo make uninstall` will remove the current version.

If we want to change this around, it should probably just be removed instead.

Reply:
Is it possible that someone could still be using Version 1? If so, maybe we should rework the readme instructions and leave the `rm` as it was.

Reply:
Removed the redundant dependency though.

Reply:
It'll always be possible that someone is still using V1, although it is unlikely.

Reply:
This change still needs to be reverted